pathspider/cmd/metadata.py

```diff
@@ -24,9 +24,12 @@ def metadata_from_ps_ndjson(fp):
             z = b    
 
     return {'_time_start': y.strftime("%Y-%m-%dT%H:%M:%SZ"),
-            '_time_end': z.strftime("%Y-%m-%dT%H:%M:%SZ")}
+            '_time_end': z.strftime("%Y-%m-%dT%H:%M:%SZ"),
+            "_file_type": "pathspider-v2-ndjson-bz2"}
```

Comment:
If the upload command should be decoupled from the PTO this line should be placed somewhere else. Maybe selectable via the parser:
`"_file_type": "pathspider-v2-ndjson-bz2"`

Reply:
I think the reason I didn't have this in the original metadata generation command I added was that PATHspider would only ever be producing metadata files for PATHspider, so it was always going to be a pathspider ndjson file type.